lib/resque/scheduler/delaying_extensions.rb

```diff
@@ -68,13 +68,10 @@ def delayed_queue_schedule_size
         redis.zcard(:delayed_queue)
       end
 
-      def next_delayed_item(before:)
-        item, time = redis.zrangebyscore(:delayed_queue, 0.0, before.to_i, limit: [0, 1], with_scores: true).first
-
-        if item
-          redis.zrem(:delayed_queue, item)
-          decode_without_nonce(item)
-        end
+      def next_delayed_item(before:, count: 1)
```

Comment:
Perhaps the method name should now be plural, since it's returning an array of items?

Reply:
OR maybe have a helper `next_delayed_item` that returns the first element from `next_delayed_items`.